### PR TITLE
Small Error Fix

### PR DESCRIPTION
**Pull request by ericxtang**
**What does this pull request do? Explain your changes. (required)**

Small fix - we were returning erorrs when a sender is not found.  This happens naturally when a new node is starting and a new Eth addr is used.

**Specific updates (required)**

Just return an empty struct instead of an error to the cli.

**How did you test each of these updates (required)**

By hand.

### CODE REVIEW

File: server/handlers.go [hunk 1]
```diff
@@ -215,8 +215,20 @@ func senderInfoHandler(client eth.LivepeerEthClient) http.Handler {
 
 		sender, err := client.Senders(client.Account().Address)
 		if err != nil {
-			respondWith500(w, fmt.Sprintf("could not query sender info: %v", err))
-			return
+			if err.Error() == "ErrNoResult" {
+				type Sender struct {
```
Comment (j0sh) [inline server/handlers.go:219]: Just curious, if the upstream Sender struct ever changes, will the compiler catch it here and prompt for an update?

**Review by j0sh — APPROVED** [review]
LGTM

File: server/handlers.go (hunk 1 above)
Comment (ericxtang) [reply server/handlers.go:219]: I think it does
Screen Shot 2019-04-01 at 7 15 18 PM

**Review by j0sh — APPROVED** [review]
Looks good - might want to rebase before merging this because there'll be an extraneous merge commit here. :ship:

**Pull request merged**